inc/core/styles/css_prop.php

```diff
@@ -273,6 +273,10 @@ public static function get_suffix( $meta, $device, $value, $css_prop ) {
 			Font_Manager::add_google_font( $font, strval( $value ) );
 		}
 
+		if ( isset( $meta[ Dynamic_Selector::META_IS_UNITLESS ] ) && $meta[ Dynamic_Selector::META_IS_UNITLESS ] ) {
+			$suffix = '';
+		}
```

Comment:
I think this can probably be added at the top as an early return. 👍🏻

Reply:
I tried an early return here but I don't think it's possible. First, we need to get the suffix. Only if it's 'em' we return an empty string. We can't check only for the 'META_IS_UNITLESS' and return early.

Reply:
Makes sense. 👍🏻  It's just that `META_IS_UNITLESS` doesn't really act according to its' name. Maybe we could use something like `META_UNITLESS_EM` or something similar instead.

Reply:
Also, we should probably also check that the suffix is `em` before we change it to an empty string. 
```suggestion
		if ( isset( $meta[ Dynamic_Selector::META_IS_UNITLESS ] ) && $meta[ Dynamic_Selector::META_IS_UNITLESS ] && $suffix === 'em' ) {
			$suffix = '';
		}
```

If I'm not mistaken, in the current state, it seems that this would apply for PX values as well.